packages/amplify-util-mock/src/api/lambda-trigger-handler.ts

```diff
@@ -0,0 +1,134 @@
+import { $TSContext } from 'amplify-cli-core';
+import { DynamoDBStreams, Endpoint } from 'aws-sdk';
+import { invokeLambda } from './lambda-invoke';
+import { isMockable } from 'amplify-category-function';
+
+/**
+ * Asynchronous function that handles invoking the given lambda trigger function
+ * when there are DynamoDB records available to be processed via DDB stream.
+ * @param context The CLI context
+ * @param streamArn ARN corresponding to the DDB stream to listen to
+ * @param lambdaTriggerName Lambda function to trigger
+ * @param localDynamoDBEndpoint Local DDB endpoint that was provisioned
+ */
+export const lambdaTriggerHandler = async (
```

Comment:
nit: please rename this to be ddbLocalStreamLambdaTrigger() if it only processes DynamoDB stream events for DDB-local.

Reply:
renamed to `ddbLambdaTrigger` since `local & stream` are kind of implied for a mock plugin